pkgs/development/tools/gomod2nix/builder/default.nix

```diff
@@ -0,0 +1,387 @@
+{ stdenv
+, stdenvNoCC
+, runCommand
+, buildEnv
+, lib
+, fetchgit
+, removeReferencesTo
```

Comment:
`trimpath` in `GOFLAGS` handles this, `removeReferencesTo` and `preFixup` isn't needed.

```suggestion
```